typing/typetexp.ml

```diff
@@ -68,8 +68,10 @@ type error =
 exception Error of Location.t * Env.t * error
 exception Error_forward of Location.error
 
+(** Map indexed by type variable names. *)
+module VarMap = Misc.StringMap
```

Comment:
Here is an example of why I recommended not having the `VarMap` alias up above in `closure.ml`: here the same alias means something completely different!  (And it's not like it's a one-letter alias.)

Reply:
I removed all instances of `VarMap` outside of the typechecker and renamed that one as `TyVarMap`, which should make things clearer.